libs/transactions/src/lib/components/transaction-item/transaction-item.component.html

```diff
@@ -9,10 +9,20 @@
   transaction.valueDate | date: 'MMM. d'
 }}</span>
 <span class="transactions-item__description">
-  <span class="transactions-item__merchant">{{
-    transaction.merchant?.name
+  <span class="transactions-item__counterParty">{{
+    transaction.counterPartyName
   }}</span>
   <span class="transactions-item__transactionType">{{ transaction.type }}</span>
+
+  <span class="transactions-item__additionalDetails">
```

Comment:
I want to challenge this in the example.
First of all because of the span compared to the div.
Secondly, it will be needed to override styles from the inside of the slot if they won't match the customers' theme.
Thirdly, if the styles will be provided on the level where we define the content of the addition slot, then it can serve as an example of how to do this.

Reply:
> First of all because of the span compared to the div.
What exactly do you mean with this?

I will investigate / add an example for overriding styles

Reply:
Journey no longer adds wrapper + styling to the extension anymore, expects app/implementation to style the extension

Moved the bundle component to separate ts/html/scss files as part of this.

Reply:
Regarding the `span` and `div` difference, I mean two things:
- that the first one is an inline-block and the second is a block
- the content model difference between the tags. `span` belongs to the `phrasing content` and for this example, it works perfectly, but as a generic example the `flow content` of the div would suit more. And because it might be missed from the reader's perspective I would vote to have the `div` if we want to have the wrapper for the additional slot.